natgeo/mortar
# [AEM-6765] Removing transform from POD at mobile breakpoint

https://jira.natgeo.com/browse/AEM-6765

Removing transform from POD at mobile breakpoint as it is unnecessary and causing problems with the kicker

To test:
Pull AEM-6765-2
gulp serve
Navigate over to http://localhost:3000/components/podpromo/
Reduce viewport to mobile breakpoint
Change "Sponsored" dropdown to false
Ensure you can hover kicker and that url changes to a kicker specific url (vs card level url)
